src/autopas/containers/linkedCells/traversals/LinkedCellTraversalInterface.h

```diff
@@ -15,15 +16,6 @@ namespace autopas {
  * @tparam ParticleCell
  */
 template <class ParticleCell>
-class LinkedCellTraversalInterface {
- public:
-  /**
-   * Traverse all pairs of cells.
-   * This function needs to be implemented by derived classes and handles to
-   * order in which the cells are traversed.
-   * @param cells Vector of cells to traverse
-   */
-  virtual void traverseCellPairs(std::vector<ParticleCell> &cells) = 0;
-};
+class LinkedCellTraversalInterface {};
```

Comment:
If there is nothing left in the Interface we might as well delete it

Reply:
The linked cells container really only requires a `CellPairTraversal`. That is why this interface is empty. We could definitely delete it. 
The reason why I left it was that I think it makes a lot of sense to have the information which traversals are allowed for this container somehow reflected in the class hierarchy. The direct sum traversal is also a `CellPairTraversal` at the moment, for example. This can be used to check whether a traversal fits to the container, like we would otherwise do purely with the `TraversalOption` and `ContainerOption`, but we also have a place to add new methods that all linked cells traversals should have in the future.

Just tell me your decision on this, and I will realize it. (I am assuming that this decision also holds for the `DirectSumTraversalInterface`.

Reply:
document use for casting